utils/downloader.py

```diff
@@ -35,30 +39,28 @@ def download_content(link: str, path: str, file_name: str):
     """
 
     with requests.get(link, stream=True) as res:
-        image_size = int(res.headers.get('content-length', 0))
-        chunk_size = 1024
-        image_content = res.iter_content(chunk_size)
+        image_size: int = int(res.headers.get('content-length', 0))
+        chunk_size: int = 1024
+        image_content: Iterator = res.iter_content(chunk_size)
 
-        with Progress() as progress:
-            download_task = progress.add_task(f"[green] {file_name}", total=image_size)
+    with Progress() as progress:
```

Comment:
Images not downloading properly. Try putting the second context manger(for progress) in the first context manager(res) because second context manager can't access the image_content properly since res is a stream.

Reply:
fixed !